app/helpers/orchid/sort_helper.rb

```diff
@@ -28,19 +28,29 @@ def sorter
   end
 
   def sort_fields_facets
+    # include "grouping" => "separator" to insert separator in dropdown
     {
       "count|desc" => "Count (most first)",
       "count|asc" => "Count (least first)",
-      "term" => "separator",
+
+      "terms" => "separator",
       "term|asc" => "Alphabetically (A-Z)",
       "term|desc" => "Alphabetically (Z-A)"
     }
   end
 
   def sort_fields_search
-    # relevancy|desc included by default with q parameter
-    # include "grouping" => "separator" to indicate breaks in dropdown
-    {
+    if params.key?("q")
+      fields = {
+        "relevancy|desc" => "Relevancy",
+        "rel_separator" => "separator"
+      }
+    else
+      fields = {}
+    end
+
+    # include "grouping" => "separator" to insert separator in dropdown
+    fields.merge({
```

Comment:
I started working on an idea like this and discarded it, though I like how you handled relevancy better.  Could we split the default fields being merged here into their own method so that to override them, you can just plug in a few fields, whereas now to override / alter the fields, you would need to copy this entire method with all the relevancy logic at the topic?

`fields.merge(additional_search_fields)` or something like that, is basically what I'm picturing